fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageUtils.java

```diff
@@ -385,15 +386,16 @@ public static String mintOCFLObjectId(final String fedoraIdentifier) {
      * @param ocflWorkDir The ocfl work directory
      * @return the repository
      */
-    public static MutableOcflRepository createRepository(final File ocflStorageRootDir, final File ocflWorkDir) {
-        ocflStorageRootDir.mkdirs();
-        ocflWorkDir.mkdirs();
+    public static MutableOcflRepository createRepository(final Path ocflStorageRootDir, final Path ocflWorkDir)
+            throws IOException {
+        Files.createDirectories(ocflStorageRootDir);
+        Files.createDirectories(ocflWorkDir);
 
         log.info("Fedora OCFL persistence directories:\n- {}\n- {}", ocflStorageRootDir, ocflWorkDir);
```

Comment:
Maybe reduce this logging to `debug`, in favor of the cleaner output of `OcflPropsConfig`.

Reply:
done